packages/resource-deployment/scripts/batch-pool-enable-msi.sh

```diff
@@ -102,9 +102,21 @@ assignSystemIdentity() {
   VMSS name: $vmssName
   System-assigned identity: $systemAssignedIdentity
 "
+
+    addResourceGroupNameTagToVMSS $vmssResourceGroup $vmssName
```

Comment:
check if you need to wait for enabling managed identity to complete. (vmss will be in updating state. during this time, if you try to make any changes you may get conflict error.

refer https://github.com/microsoft/accessibility-insights-service/pull/566/commits/4f49f574625749103c821a487407d97c540d0c79